internal-log-fetcher/mina-graphql-client/src/main.rs

```diff
@@ -151,6 +302,66 @@ async fn main() -> Result<()> {
             println!("authorized");
             client.schedule_zkapp_payments(cmd.into()).await?;
         }
+        Command::SchedulePayments(cmd) => {
+            client.authorize().await?;
+            println!("authorized");
```

Comment:
Followup: It seems like we are already using print statements in this codebase, but I would suggest integrating an actual logger. 

I can help with this if this would be useful